### PR TITLE
Update Readme.txt for 1.84

### PR DESCRIPTION
**Pull request by scotthibbs**
Added Verbage

### CODE REVIEW

**Comment by scotthibbs** [comment]
I'm rewriting the Readme.txt. Are there any more incorrect entries for the log window? 
I have this: 
LOG WINDOW
  The log window marks incorrect entries in your log as follows:
  DUP   - duplicate QSO.
  NIL   - not in other station's log: you made a mistake in the call sign, 
		  or waited to long to save the information. 
  RST   - incorrect RST in your log.
  NR    - incorrect exchange number in your log.
  CL    - incorrect Arrl Field Day Classification in your log.
  NAME  - incorrect Name in your log.
  SEC   - incorrect ARRL Section in your log.
  ST    - incorrect State in your log.

Am I missing any new ones?

**Comment by w7sst** [comment]
> DUP   - duplicate QSO.
>   NIL   - not in other station's log: you made a mistake in the call sign,
>  or waited to long to save the information.
>   RST   - incorrect RST in your log.
>   NR    - incorrect exchange number in your log.
>   CL    - incorrect Arrl Field Day Classification in your log.
>   NAME  - incorrect Name in your log.
>   SEC   - incorrect ARRL Section in your log.
>   ST    - incorrect State in your log.
>
> Am I missing any new ones?

Looking at the code, the following are reported:
```
                Err := 'QTH';
                Err := IfThen(IsNum(TrueExch2), 'ZN ', 'Soc');
                Err := 'ERR';
        etCqZone:      if TrueNr <> NR then Err := 'ZN ';
        etArrlSection: if TrueExch2 <> Exch2 then Err := 'SEC';
        etItuZone:     if TrueExch2 <> Exch2 then Err := 'ZN ';
```

So I would add:
- QTH - Incorrect location information.
- ZN - Incorrect CQ-Zone or ITU-Zone information
- Soc - Society information is wrong (JARL)
- Sec - Incorrect ARRL Section

For ST above, please modify to include State/Provence

Thank you.

**Comment by scotthibbs** [comment]
Added the new log errors - thanks Mike!

**Comment by scotthibbs** [comment]
Fixed wording for the run button.. I think I got it. Please check me.

**Review by w7sst — APPROVED** [review]

**Comment by w7sst** [comment]
Hi @scott,
I merged the conflicts in Readme.txt after I merged my last change to mainline. Please review my changes. Thanks.

**Comment by w7sst** [comment]
@scotthibbs ,
Hi Scott, I notice that the `Readme.txt` contains tab characters. This can mess up formatting on some screens. We should try to get these tabs removed.

**Comment by scotthibbs** [comment]
Tabs removed. :)

File: Readme.txt
Comment (w7sst) [inline Readme.txt:264]: SOC - this is for the new IARU HF Contest. This is an error in the IARU-Society information. The JARL contests report NR as exchange errors.

**Review by w7sst — CHANGES_REQUESTED** [review]
I'll approve again after a few minor changes.

**Comment by w7sst** [comment]
@scotthibbs , I added some additional items to the `Readme.txt` file to finish out integrating f6fvy's work. Keep me posted on your progress.

**Comment by scotthibbs** [comment]
I tried to simplify the document and changed some spacing to make it look "pretty".

**Comment by scotthibbs** [comment]
Wrong place for this: 
Dangit. I removed the What's Changed section of the draft release. Then I noticed that I left this in for the other releases. I don't think this is critical, but if you want to "redo" the draft release with my changes (basically changed the install instructions), it should be ready to fly.

File: Readme.txt
Comment (scotthibbs) [reply Readme.txt]: Changed this to read:
"You may have to hit F1 (or Enter) to start sending CQ."

**Comment by w7sst** [comment]
@scotthibbs 
Hi Scott, I discovered that this branch was corrupted somehow (couldn't figure out how this occurred). Anyway, I applied a correction patch to fix it. I then removed trailing whitespace and adjusted the indentation for the new IARU HF Contest description.

I think this is ready for me to merge now. We can always make additional changes.  Tnx, Mike

File: Readme.txt
Comment (f6fvy) [inline Readme.txt:58]: "the key" is repeated twice (lines 57 and 58)

File: Readme.txt
Comment (f6fvy) [inline Readme.txt:175]: "F2" instead of "f2" for consistency

File: Readme.txt
Comment (f6fvy) [inline Readme.txt:177]: "F2" instead of "f2" for consistency

File: Readme.txt
Comment (f6fvy) [inline Readme.txt:224]: Alt-F9 and Alt-F10 also work to adjust the keying speed

File: Readme.txt
Comment (scotthibbs) [reply Readme.txt]: Thanks!

File: Readme.txt
Comment (scotthibbs) [reply Readme.txt:224]: I removed these because my Alt-F9 in my windows 10 displays my GeForce Experience program asking me to turn on desktop capture. Alt -F10 does nothing. Is this just my PC or this release? 

(The other two Alt commands gives me the dreaded windows chime - I'd remove them if I could). 

Please check this for me and let me know.. Thanks, Scott

Comment (w7sst) [reply Readme.txt:224]: Yes, we should add the Alt-F9 and Alt-F10. These choices are still in the code and work on my laptop. Tnx, Mike.
@scotthibbs

Comment (w7sst) [reply Readme.txt:224]: > I removed these because my Alt-F9 in my windows 10 displays my GeForce Experience program asking me to turn on desktop capture. Alt -F10 does nothing. Is this just my PC or this release?

I would guess your Alt-F9 is mapped for your GeForce program. Thankfully MRCE has many key mappings so you can choose the one(s) that work for you.

Comment (w7sst) [reply Readme.txt:224]: Add Alt-F9/Alt-F10 to `Readme.txt` file as suggested.

**Comment by w7sst** [comment]
@scotthibbs Are we ready to merge this now? We can still make additional changes if needed. Tnx, Mike

**Comment by scotthibbs** [comment]
Yes. I was waiting on an answer about alt-F9 and alt-F10. I just tested and it worked on one of my three computers. So if it is added to the readme, I'm a go for launch!

**Pull request merged**
